package.json

```diff
@@ -2,14 +2,54 @@
   "name": "code-exercise",
   "version": "1.0.0",
   "description": "This is a shopping cart coding exercise",
-  "main": "index.js",
+  "main": "main.ts",
+  "index": "index.html",
   "scripts": {
+    "ng": "ng",
+    "dev": "ng serve",
+    "tst": "ng test",
+    "build": "ng build",
+    "e2e": "ng e2e",
+    "server": "canned -p 5000 ./server/",
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "canned -p 5000 ./server"
   },
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "canned": "^0.3.12"
+    "@angular/animations": "~11.2.1",
+    "@angular/common": "~11.2.1",
+    "@angular/compiler": "~11.2.1",
+    "@angular/core": "~11.2.1",
+    "@angular/forms": "~11.2.1",
+    "@angular/platform-browser": "~11.2.1",
+    "@angular/platform-browser-dynamic": "~11.2.1",
+    "@angular/router": "~11.2.1",
+    "bootstrap": "^4.6.0",
+    "canned": "^0.3.12",
+    "font-awesome": "^4.7.0",
+    "jquery": "^3.5.1",
```

Comment:
Please drop `jQuery`...If it's just for the carousel, please use some Angular-specific carousel plugin

Reply:
done